### DISCUSSION TITLE
hassos 11.3rc2 issues

### DISCUSSION THREAD

**Discussion by scooper1** [blakeblackshear/frigate#9196] [General]
just upgraded from 11.3rc2   
issue with recordings on hassos network share for frigate recordings 
frigate addon beta 7 full access
the storage and system menu pages are causing a error when asking for stats on the network share
this is causing a stall on the share storage  so recordings are failing 
this also happens when frigate is checking the storage used 
11.3.rc1 was ok 
11.3.rc2  makes changes to network tables and routing  which seems to be a problem 
not raised as an issue as maybe fixed in 11.3.rc3  or could just be my system
the share is provided by truenas

**Comment by ovizii** [comment]
Seems to be indeed an issue, see this issue: https://github.com/blakeblackshear/frigate/issues/9218 - I didn't check discussions, only issues before opening mine. Just thought I'd loop you in.
